### DISCUSSION TITLE
How does the extra point in PA4 work?

### DISCUSSION THREAD

**Discussion by Nayoung-Oh** [ANLAB-KAIST/KENSv3#87] [Project Q&A 4. Etc] [answered]
I have a question about the extra point we can get in PA4. If we get the full point in PA4 and also finish the extra point cases, is the point exceed 100% and added to the other PA score..? Or if we get the full score, the extra case points do not affect our PA score?
Thank you in advance!!

**Comment by Pusnow** [comment, marked as answer]
It does not affect other PA scores.

**Reply by Nayoung-Oh** [reply]
Oh, so if I all pass the basic cases, I cannot get more scores by implementing extra cases.... Thank you for reply!
